docker-for-mac/index.md

```diff
@@ -297,15 +297,23 @@ choose to discard or not apply changes when asked.
 
 ### Kubernetes
 
-**Kubernetes is only available if you are part of the private beta for Docker for Mac 17.12. To access beta builds, you must be signed in within Docker for Mac using your Docker ID.**
+**Kubernetes is only available in the private beta of Docker for Mac 17.12 CE Edge through the [Docker Beta program](https://beta.docker.com/). To access beta builds, you must be signed in with your Docker ID within Docker for Mac: select ![whale menu](/docker-for-mac/images/whale-x.png){: .inline} -> Sign in / Create Docker ID from the menu bar.**
```

Comment:
...to participants in the private beta.... (maybe?) The deal is that 17.12 CE Edge is available to everyone, but everyone won't be able to enable Kubernetes, I think. Maybe verify that.

Reply:
@mistyhacks , are you saying that this is not enough? 
> Kubernetes is only available in the private beta of Docker for Mac 17.12 CE Edge through the [Docker Beta program](https://beta.docker.com/).

Reply:
The section just below could use the clarification. Perhaps that is what you meant.

Reply:
It just reads weird to me. I think it is only available in this specific D4M build which you can only access if you are in the private beta program. Not sure how to convey that.